### ISSUE TITLE
Console Errors Spamming when players join server

### ISSUE THREAD

**Issue by PiggiesGoSqueal** [Hazebyte/CrateReloaded#596]
Getting tons of console error spam. 

Version:  
This server is running Paper version git-Paper-280 (MC: 1.20.2) (Implementing API version 1.20.2-R0.1-SNAPSHOT) (Git: 4675152)
You are running the latest version

Plugin ver: CrateReloaded v2.3.1 (free)
latest.log: https://hastebin.skyra.pw/ociwixomid.md

```[06:20:03 INFO]: UUID of player leilamary is a4e659d3-be19-4b52-8a63-01238d7804e4
[06:20:03 INFO]: [CrateReloaded] [ACF] Can't read players locale, you will be unable to automatically detect players language. Only Bukkit 1.7+ is supported for this.
[06:20:03 INFO]: [CrateReloaded] [ACF] java.lang.NoSuchFieldException: locale
[06:20:03 INFO]: [CrateReloaded] [ACF]  at java.base/java.lang.Class.getDeclaredField(Class.java:2642)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at CrateReloaded.jar//com.hazebyte.libs.aikar.acf.BukkitCommandManager.readPlayerLocale(BukkitCommandManager.java:324)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at CrateReloaded.jar//com.hazebyte.libs.aikar.acf.ACFBukkitListener.onPlayerJoin(ACFBukkitListener.java:57)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:40)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:77)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:126)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:615)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at net.minecraft.server.players.PlayerList.a(PlayerList.java:346)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at net.minecraft.server.network.ServerConfigurationPacketListenerImpl.a(ServerConfigurationPacketListenerImpl.java:130)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at net.minecraft.network.protocol.configuration.ServerboundFinishConfigurationPacket.a(ServerboundFinishConfigurationPacket.java:18)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at net.minecraft.network.protocol.configuration.ServerboundFinishConfigurationPacket.a(ServerboundFinishConfigurationPacket.java:9)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$ensureRunningOnSameThread$0(PlayerConnectionUtils.java:53)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at net.minecraft.server.TickTask.run(TickTask.java:18)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at net.minecraft.util.thread.IAsyncTaskHandler.d(IAsyncTaskHandler.java:153)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.d(IAsyncTaskHandlerReentrant.java:24)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1324)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:193)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at net.minecraft.util.thread.IAsyncTaskHandler.x(IAsyncTaskHandler.java:126)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at net.minecraft.server.MinecraftServer.bg(MinecraftServer.java:1301)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1294)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at net.minecraft.util.thread.IAsyncTaskHandler.c(IAsyncTaskHandler.java:136)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at net.minecraft.server.MinecraftServer.u_(MinecraftServer.java:1272)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1160)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:315)
[06:20:03 INFO]: [CrateReloaded] [ACF]  at java.base/java.lang.Thread.run(Thread.java:833)
```

Feedback would be greatly appreciated.

**Comment by PiggiesGoSqueal** [comment]
Bump

**Comment by imWillX** [comment]
Will be fixed in 2.3.8
